src/app/screens/form-register/form-register.component.ts

```diff
@@ -12,22 +14,32 @@ export class FormRegisterComponent implements OnInit {
   post: any;
   messageAlert: string = 'Campo obligatorio'
 
-  constructor( private fb: FormBuilder ) { }
+  constructor( private fb: FormBuilder, private registerService: RegisterService, private getdataService: GetdataService) { }
```

Comment:
Getdata Should be in camelcase (GetDataService),  by the way, try to name the services with an specific name